doc/rst/technotes/gpu.rst

```diff
@@ -97,10 +86,106 @@ kernels for the ``forall`` loops in the function.
     }
   }
 
+Setup and Compilation
+---------------------
 
-Multi-Locale Support
+To enable GPU support set the environment variable ``CHPL_LOCALE_MODEL=gpu``
+before building Chapel. Chapel's build system will automatically try and deduce
+where your installation of CUDA exists. If the build system fails to do this,
+or you would like to use a different CUDA installation, you can set
+``CHPL_CUDA_PATH`` environment variable to the CUDA installation root.
+
+We also suggest setting ``CHPL_RT_NUM_THREADS_PER_LOCALE=1`` (this is necessary
+if using CUDA 10).
+
+To compile a program simply execute ``chpl`` as normal. By default the generated
+code will target compute capability 6.0 (specifically by passing
+``--cuda-gpu-arch=sm_60`` when invoking clang). If you would like to target a
+different compute capability (necessary for example, when targeting Tesla K20
+GPUs) you can pass ``--gpu-arch`` to ``chpl`` and specify a different value
+there.  This may also be set using the ``CHPL_CUDA_ARCH`` environment variable.
+
+If you would like to view debugging information you can pass ``--verbose`` to
+your generated executable. This output will show the invocation of CUDA kernel
+calls along with various other interactions with the GPU such as memory
+operations.  You may also use the :mod:`GPUDiagnostics` module to gather
+similar information.
+
+Requirements and Limitations
+----------------------------
+
+Because of the early nature of the GPU support project there are a number of
+limitations.  We intend to lift many of these limitations in future releases
+but the following list (while not exhaustive) identifies what we do not
```

Comment:
I don't think the "while" is necessary on this line (what's the "but"? usually it's "while X, [something despite X]")